fontforge/othersubrs.c

```diff
@@ -526,8 +526,11 @@ return( false );
     }
     fclose( os );
     /* we just read a copyright notice? That's no use */
-    if ( sub_num<=0 )
+    if ( sub_num<=0 ) {
+	free(co);
+	free(lines);
 return( false );
+    }
```

Comment:
I don't envy you this one; it's a mess. So I'll be less didactic. 

`co`:
```
for ( i=0; co[i]!=NULL; i++)
    free(co[i]);
free(co)
```
`lines`:
```
for ( i=0; i<l; i++)
    free(lines[i]);
free(lines);
```
Both of these should occur before `return` above, and the `lines` part should go below where `free(lines)` is right now on 543. (I don't actually know if `l` can be `> 0` coming out of that loop, but the loop itself doesn't imply that it can't.)

Reply:
Did too.